src/typeDefs/types.ts

```diff
@@ -375,6 +375,7 @@ export const types = gql`
     educationGrade: EducationGrade
     email: EmailAddress!
     employmentStatus: EmploymentStatus
+    post: [Post]
```

Comment:
`posts(after: String, before: String, first: Int, last: Int): PostsConnection`

you have to make use of a graphql connection, you can't just return a full blown list of objects all at once, I had mentioned this in the issue

you need to care about naming things correctly, this is standard practice

Reply:
@AVtheking yes, cursor based pagination as is the standard for graphql

Reply:
Ok will learn and implement it